schema.sql

```diff
@@ -1,12 +1,24 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    id INTEGER,
+    id SERIAL PRIMARY KEY,
     name VARCHAR(255),
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL(5,2)
-    species VARCHAR(255)
+    weight_kg DECIMAL(5,2),
+    species_id INTEGER REFERENCES species(id),
+    owner_id INTEGER REFERENCES owners(id)
```

Comment:
[OPTIONAL] Removing the `species VARCHAR(250)` and adding the `species_id...` is not the best way to alter or making changes to the table. Kindly consider using correct sql commands to remove and add changes later to achieve optimum result. 
For instance, you can use the following commands: 

      ALTER TABLE REMOVE species; (to remove column species)
      ALTER TABLE animals ADD species_id int REFERENCES species(id); (to add species id)
      ALTER TABLE animals ADD owner_id int REFERENCES owners(id); (to add owners id)